Jenkinsfile

```diff
@@ -326,7 +329,7 @@ pipeline {
                 TAG = "${env.VERSION}-${env.GIT_COMMIT}"
                 DOCKER_GCR = credentials("zeebe-gcr-serviceaccount-json")
                 ZEEBE_AUTHORIZATION_SERVER_URL = 'https://login.cloud.ultrawombat.com/oauth/token'
-                ZEEBE_CLIENT_ID = 'W5a4JUc3I1NIetNnodo3YTvdsRIFb12w'
+                ZEEBE_CLIENT_ID = 'ELL8eP0qDkl6dxXVps0t51x2VkCkWf~p'
```

Comment:
I don't think this will work for `lastestStableBranchName="stable/0.26"` I am not sure we need a concept like `latestStableBranchName`. We could just hard code it for develop and then hardcode it in the stable branches. In that case we could rename `isDevelopBranch` to `isStableBranch`

However, if we want to have a latest stable branch, then the client id, secret needs to depend on the branch.

Or maybe I am missing something. Happy to discuss it f2f.

Reply:
I understand. How about we change `latestStableBranchName` to `stable/1.0`? Then I think should immediately work. As soon as develop starts to divert from `stable/1.0` we need to create a new orchestration cluster for develop.